examples/model_selection/plot_roc.py

```diff
@@ -437,7 +439,19 @@
 # global performance of a classifier can still be summarized via a given
 # averaging strategy.
 #
-# Micro-averaged OvR ROC is dominated by the more frequent class, since the
-# counts are pooled. The macro-averaged alternative better reflects the
-# statistics of the less frequent classes, and then is more appropriate when
-# performance on all the classes is deemed equally important.
+# When dealing with imbalanced datasets, choosing the correct metric is crucial.
+# Accuracy may not always reflect model performance well in these cases.
+# For example, metrics such as F1 score, precision, or recall might be more
+# appropriate, depending on the business context or problem you are addressing.
+# It is also essential to select an appropriate averaging method (micro vs. macro)
+# depending on the desired outcome:
+#
+# - Micro-averaging aggregates metrics across all instances, treating each
+#   individual instance equally, regardless of its class. This approach is useful
+#   when evaluating overall performance, but note that it can be dominated by
+#   the majority class in imbalanced datasets.
+#
+# - Macro-averaging calculates metrics for each class independently and then
+#   averages them, giving equal weight to each class. This is particularly useful
+#   when you want under-represented to be considered as important as highly
+#   populated classes.
```

Comment:
I'm pretty happy with this part. Basically, if we still use a standard statistical approach then this is nice to understand what this averaging means.